lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -172,6 +172,8 @@ def video_summary(video_profiles, course_id, video_descriptor, request, local_ca
         "only_on_web": video_descriptor.only_on_web,
     }
 
+    all_sources = []
```

Comment:
Curious (side question): do we know if there's still a need to update this deprecated API?  That is, do you know of clients that are still using this API?

Reply:
@nasthagiri The [edx-app-ios code is still using it](https://github.com/edx/edx-app-ios/blob/a4ec26b3bb90a578678885cd541a93a0b6ab8afc/Source/OEXNetworkConstants.h#L28).

I was surprised by this too..  I tried all the combinations (update just course_blocks, update just mobile_api, update both), and came to the conclusions described in the PR description "Author Notes & Concerns"

Reply:
@pomegranited oops.  I missed that note in the Author Notes & Concerns.

We do have this open JIRA ticket for removing the call from the Android app: https://openedx.atlassian.net/browse/LEARNER-2764

I would love to remove all calls to this old endpoint so we can finally remove that code.

Reply:
+1 to that..  Thanks for tying up that thread on the LEARNER ticket.